charts/hono/templates/hono-listPods-role.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $args := dict "dot" . "name" "listPods" }}
```

Comment:
is there a particular reason why this role definition is not in the command router folder as well?